webpack/components/extensions/HostDetails/Tabs/RemoteExecutionActions.js

```diff
@@ -209,6 +209,16 @@ export const removePackage = ({ hostname, packageName }) => post({
   errorToast,
 });
 
+// Used by packages wizard
```

Comment:
In `PackagesTab.test.js` I was surprised to find there are no tests for remove. Could we add some? Rather than just a comment here, I think tests are a better way to ensure this doesn't happen in the future. They should be easy to michaelangelo from the other tests..

Reply:
Added a test for both bulk and single kebab remove